### ISSUE TITLE
php-fpm 'httpd entered FATAL state, too many start retries too quickly'

### ISSUE THREAD

**Issue by 0x0256c03a** [Leantime/leantime#908]
Hi ! I do have an issue using leantime + docker. I am using the offical docker-compose file just a little tweaked.
The configuration seems to work and the container continues to run however nothing seems to run and by looking at the logs I see that fpm seems in a hurry....

![image](https://user-images.githubusercontent.com/106661682/186172020-b3395f7c-6cb1-49a7-ab67-43cc51ea0545.png)

```yml

leantime:
    image: leantime/leantime:latest
    container_name: leantime
    restart: unless-stopped
    volumes:
      - /docker/leantime/public_userfiles:/var/www/html/public/userfiles
      - /docker/leantime/userfiles:/var/www/html/userfiles

```
The environment configuration is in a .env file.

I've seen the same issue before but don't seems to work:
https://stackoverflow.com/questions/56889046/supervisor-celery-entered-fatal-state-too-many-start-retries-too-quickly

Any help if someone encountered the same problem ?

**Comment by alexo5190** [comment]
Yes I am have the same issues with a similar docker compose.

**Comment by ecker00** [comment]
Trying out LeanTime for the first time, and also experiencing this.

**Comment by marcelfolaron** [comment]
Hi All,

Sorry there was an error in the sample nginx config. The redirect rules need to be:

```
rewrite ^/?$ /index.php?act=dashboard.show;
    rewrite ^/([^/\.]+)/?$ /index.php?act=$1;
    rewrite ^/([^/\.]+)/([^/\.]+)/?$ /index.php?act=$1.$2;
    rewrite ^/([^/\.]+)/([^/\.]+)/([^/\.]+)/?$ /index.php?act=$1.$2&id=$3;
```

**Comment by ecker00** [comment]
I tried to change `docker-compose.yml` to use `image: leantime/leantime:2.2.3` (stepped down one version at the time), but all of them that image logging:
```
leantime        | 2022-08-29 20:25:33,758 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
leantime        | 2022-08-29 20:25:33,772 INFO supervisord started with pid 11
leantime        | 2022-08-29 20:25:34,804 INFO spawned: 'httpd' with pid 14
leantime        | 2022-08-29 20:25:34,818 INFO spawned: 'php-fpm' with pid 16
leantime        | 2022-08-29 20:25:34,935 INFO exited: httpd (exit status 1; not expected)
leantime        | [29-Aug-2022 20:25:34] NOTICE: fpm is running, pid 16
leantime        | [29-Aug-2022 20:25:34] NOTICE: ready to handle connections
leantime        | 2022-08-29 20:25:35,988 INFO spawned: 'httpd' with pid 24
leantime        | 2022-08-29 20:25:35,990 INFO success: php-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
leantime        | 2022-08-29 20:25:36,110 INFO exited: httpd (exit status 1; not expected)
leantime        | 2022-08-29 20:25:38,135 INFO spawned: 'httpd' with pid 27
leantime        | 2022-08-29 20:25:38,256 INFO exited: httpd (exit status 1; not expected)
leantime        | 2022-08-29 20:25:41,300 INFO spawned: 'httpd' with pid 30
leantime        | 2022-08-29 20:25:41,420 INFO exited: httpd (exit status 1; not expected)
leantime        | 2022-08-29 20:25:42,433 INFO gave up: httpd entered FATAL state, too many start retries too quickly
```

The MySQL image seems happy.

**Comment by marcelfolaron** [comment]
Can someone try to run 2.1.9 and see if that works? I have a hard time reproducing this issue

**Comment by marcelfolaron** [comment]
Okay, I can finally reproduce the issue on my M1 Mac as well. Will test and get back

**Comment by marcelfolaron** [comment]
I just pushed a new image to dockerhub. (tag latest) please try and let me know if that works.

**Comment by ecker00** [comment]
I brought everything down `docker-compose down -v` (deletes databases!), I pulled changes from master and started it up again, and it works for me! 🎉

**Comment by marcelfolaron** [comment]
Thanks. Will close this for now

**Comment by mrbbbbbbbb** [comment]
This one (or 1010) should be reopened. It keeps happening now on 3.0.7, so annoying.
Leantime is basically unusable lightly speaking, added a lot of data, projects, tasks Nth time now (removed volumes a number of times) and it keeps happening worked fine 2 days and again same story, restarting docker containers does not help. Damn error below
INFO gave up: httpd entered FATAL state, too many start retries too quickly.
2 years later and still not fixed

**Comment by hermann-san** [comment]
I had the same issue today with version 3.1.4
`INFO gave up: httpd entered FATAL state, too many start retries too quickly.`
right before a reboot of the server it still worked. So it must have been some issue with the reboot. Maybe a not proper stopping of the docker container. 
I was not able to fix it, so I did restore my backup from last night and that worked. 
So it appears to be connected to an ungraceful stop of a container. I did a reboot of the Linux server without stopping the container manually.  Up until now that hasn't  caused any issues, but this time it seems to be the root cause. I don't know how to fix that so I went for the easy solution (restore of a backup). I guess one has to try to fix the issue inside the docker container.
